src/main/java/pl/psnc/dei/service/UrlBuilder.java

```diff
@@ -29,32 +31,55 @@ public String getBaseUrl() {
 	}
 
 	public String urlForAllProjects() {
-		return transcriptionPlatformLocation + ALL_PROJECTS_SUFFIX;
+		return transcriptionPlatformLocation + PROJECTS_SUFFIX;
 	}
 
-	public String urlForSendingRecord() {
-		return transcriptionPlatformLocation + RECORDS_SUFFIX;
+	public String urlForSendingRecord(Record record) {
+	    String url = transcriptionPlatformLocation
+                + PROJECTS_SUFFIX
+                + '/'
+                + record.getProject().getProjectId()
+                + STORIES_SUFFIX
+                + '?'
+                + IMPORT_NAME_PARAM
+                + record.getAnImport().getName();
+		if (record.getDataset() != null) {
+            url += '&'
+                    + DATASET_PARAM
+                    + record.getDataset().getDatasetId();
+        }
+		return url;
 	}
 
     public String urlForTranscription(Transcription transcription) {
-        return transcriptionPlatformLocation + TRANSCRIPTIONS_SUFFIX + "/" + transcription.getTp_id();
+        return transcriptionPlatformLocation + ENRICHMENTS_SUFFIX + "/" + transcription.getTp_id();
     }
```

Comment:
This endpoint might not be needed anymore, I haven't checked it while implementing changes here. Please review the code and change or remove it. Reference: https://docs.google.com/document/d/12bq8hoMgF3BbO3h1mfQCQPY_7Q6Q8IpZ6y2UieyB5Wc/edit#heading=h.m6nepqv85jqr

Reply:
isn't that a 5. point from the doc?

Reply:
Actually the comment should be done for the next method urlForTranscriptionUpdate. This one is fine...